LICENSE

```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2016-2016, atoum contributors.
 Copyright (c) 2010-2016, Frédéric Hardy.
```

Comment:
@mikaelrandy yes. If we were to change the license we would have to ask all contributor if they are OK with that.

Moreover, removing the name of @mageekguy from the license does not seem good to me: he was the copyright owner. We just choose to change this from now on.